### DISCUSSION TITLE
Visual aids/slides for Instructor Training workshops?

### DISCUSSION THREAD

**Discussion by JCSzamosi** [carpentries/trainers#169] [Instructional Content]
A theme that I've seen in feedback of the Instructor Trainings that I've done is that trainees want more visual aids for the non-live-coding parts. 

I wonder if our policy against slides, which makes sense in the technical workshops, is backfiring a bit in IT workshops where there is a lot of time spent just talking. I know some people have visual aids that they've collected over time to help with different sections of the workshop, and I'm wondering if it might make sense to create collections of visual aids associated with different sections (or, more ambitiously, a default set of slides) that we keep on the curriculum repository so people can easily draw from them.

**Comment by JCSzamosi** [comment]
It has been brought to my attention that a number of visual aids can be found [here](https://github.com/carpentries/instructor-training/tree/gh-pages/fig), and that many of them are linked from the etherpad workshop template [here](https://pad.carpentries.org/ttt-template)

**Comment by brownsarahm** [comment]
We do have a set of default slides, but we don't have a good version-control based format for them. I added the link to that on the bookmarks of the trainer slack, because I'm not sure of the permission levels on it for now. 

However, more maintainable solutions to this are still an open issue and great discussion point. 

There is also a [related issue](https://github.com/carpentries/instructor-training/issues/1000) on the IT repo with some discussion and ideas.
